### PR TITLE
Base: Add red cursor theme

### PR DESCRIPTION
**Pull request by electrikmilk**
This adds a red cursor theme.

Screenshot 2022-11-06 at 21 29 31

Screenshot 2022-11-06 at 21 29 36

Screenshot 2022-11-06 at 21 29 41

Screenshot 2022-11-06 at 21 30 02

### CODE REVIEW

File: Base/res/cursor-themes/Red/Config.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Disallowed=disallowed.png
+Move=move.png
+Hand=hand.x8y4.png
+Help=help.x1y1.png
+Drag=drag.png
+DragCopy=drag-copy.x7y7.png
+Wait=wait.f14t100.png
+Crosshair=crosshair.png
+Eyedropper=eyedropper.x2y2.png
+Zoom=zoom.x0y0.png
```
Comment (timschumi) [inline Base/res/cursor-themes/Red/Config.ini:20]: This looks like it's missing a newline at the end.

**Review by timschumi — CHANGES_REQUESTED** [review]
Those files look awfully large, please run them through `optipng` if you haven't done so already (if I recall correctly we usually use `optipng -strip all`). Alternatively, if you want to be adventurous, `zopflipng` has been in discussion lately for achieving better results than `optipng` (but please make sure that it strips all metadata and that the result is still loadable by our PNG parser).

**Review by timschumi — COMMENTED** [review]
I likely wouldn't use it personally, but at least there doesn't seem to be anything objectively wrong with it.

**Comment by ADKaster** [comment]
I do wonder, would these cursors be better suited for the serenity-theming repository? @Xexxa  what do you think?

**Comment by Xexxa** [comment]
> I do wonder, would these cursors be better suited for the serenity-theming repository? @Xexxa  what do you think?

I can't be objective on this one. What's suited for the main repo seems to be a matter of taste and I'm happy I don't have to make that decision...

**Pull request closed without merging**